lib/commands/crafter/add_office.rb

```diff
@@ -5,6 +5,9 @@
 
 module Commands
   class AddOffice
+    def self.description
```

Comment:
Shall we do a verification in `lib/commands/help.rb` if the function description exist? As the function could be optional. It feels a bit odd to me to obligate command to have it or make the `help` command explode if it is not the case

Reply:
how about?

```
class Command
  def description
     # optional description
  end
end

class MyCommand < Command
end
```

Reply:
:poop: ☝️